src/student.js

```diff
@@ -0,0 +1,8 @@
+class Student {
+  constructor (firstName, lastName, gitHub) {
+    this.fullName = `${firstName} ${lastName}`
```

Comment:
Nice way for adding the full name and last name

Reply:
same as above, i didn't think of doing that but i like the idea